controller/controller.go

```diff
@@ -137,12 +137,22 @@ func (r *VirtualServicePatchReconciler) Reconcile(ctx context.Context, request r
 		//trigger event
 		r.EventRecorder.Event(patch, "Warning", "ReconciliationFailed", fmt.Sprintf("VirtualServiceMerge reconcile error: %s", err.Error()))
 		if err2 := r.Context.Client().Get(ctx, request.NamespacedName, patch); err2 != nil {
+			if kerr.IsNotFound(err2) {
+				// do not need to panic just log output
+				r.Context.Logger().Info("Virtual service merge not found. No status to update.")
```

Comment:
should we add details of what was now found?

Reply:
don't think so, it is kind of standards to always check for `not found error ` when writing controller codes. It was a mistake on my side that I forgot to add the check.

Reply:
no harm in printing out the which virtual service is involved when this not found error happens though ,right? +1